onfido-java/src/main/java/com/onfido/EvidenceFileManager.java

```diff
@@ -0,0 +1,23 @@
+package com.onfido;
+
+import com.onfido.api.Config;
+import com.onfido.api.FileDownload;
+import com.onfido.api.ResourceManager;
+import com.onfido.exceptions.OnfidoException;
+import okhttp3.OkHttpClient;
+
+public class EvidenceFileManager extends ResourceManager {
+
+    protected EvidenceFileManager(Config config, OkHttpClient client) {
+        super("evidence_file", config, client);
+    }
+
+    public FileDownload workflowRun(String workflowRunId) throws OnfidoException {
+        return downloadRequest("?workflow_run_id=" + workflowRunId);
+    }
+
+    public FileDownload check(String checkId) throws OnfidoException {
```

Comment:
The evidence file should only be supported for workflow runs as per our [public documentation](https://documentation.onfido.com/#etsi-evidence), can you please remove this method? Thanks